disnake/user.py

```diff
@@ -162,12 +166,27 @@ def banner(self) -> Optional[Asset]:
         .. versionadded:: 2.0
 
         .. note::
+
             This information is only available via :meth:`Client.fetch_user`.
         """
         if self._banner is None:
             return None
         return Asset._from_banner(self._state, self.id, self._banner)
 
+    @property
+    def avatar_decoration(self) -> Optional[Asset]:
+        """Optional[:class:`Asset`]: Returns the user's avatar decoration asset, if available.
+
+        .. versionadded:: 2.8
+
+        .. note::
+
+            This information is only available via :meth:`Client.fetch_user`.
```

Comment:
This doesn't apply here (anymore?), the `avatar_decoration` field is sent in all places as far as I can tell. (correct me if I'm wrong)

The only edge case is disabling the member cache, which results in #870 v2; i.e. the `avatar_decoration` field should be added to `BaseUser._to_minimal_user_json`, such that `Member.avatar_decoration` works in all cases.

Reply:
it looks like `avatar_decoration` is optional though.
![image](https://user-images.githubusercontent.com/67214928/209358081-e2c7a0e0-98b2-48e7-b5b2-f6568ba6de27.png)

Reply:
Well, we can assume that `avatar_decoration` will always be sent. If it's not, then we can change this docimentation in the future.

Reply:
Removed the note in adcabe09962f336415d41dc927d67ee02e4b0886, but I will keep this conversation open so other people can also see this.

Reply:
> The only edge case is disabling the member cache, which results in #870 v2; i.e. the `avatar_decoration` field should be added to `BaseUser._to_minimal_user_json`, such that `Member.avatar_decoration` works in all cases.

Fixed in 2304d382a355308b7d6cad6bf0005838335f3d7e :)